catalog/spatial/registry/registry-federation-admin-service-impl/pom.xml

```diff
@@ -95,6 +95,13 @@
             <artifactId>filter-proxy</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <!-- Upgrade mockito to version 2.1.0 for this module to get final mock capability -->
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-core</artifactId>
+            <version>2.1.0</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
@lessarderic We should probably have a strategy for overriding the managed version of a library. What are your thoughts?